tests/c/simple_swt.c

```diff
@@ -0,0 +1,39 @@
+// Run-time:
```

Comment:
Do we need this test? Isn't it covered in other tests that don't specify what the tracer is (so will be run in hwt or swt mode)?

Reply:
Removed 👉 57aed88f9a8139d1f84651e3b83005de7373e86a